roundabout/functions/presigned-url.js

```diff
@@ -0,0 +1,69 @@
+import * as Sentry from '@sentry/serverless'
+import { S3Client } from '@aws-sdk/client-s3'
+import { CID } from 'multiformats/cid'
+
+import { getSigner } from '../index.js'
+import { getEnv } from '../utils.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+/**
+ * AWS HTTP Gateway handler for GET /presigned-url/{cid}
+ * A presigned URL for the given CID is 
+ *
+ * @param {import('aws-lambda').APIGatewayProxyEventV2} request
+ */
+export async function presignedUrlGet(request) {
+  const {
+    BUCKET_ENDPOINT,
+    BUCKET_REGION,
+    BUCKET_ACCESS_KEY_ID,
+    BUCKET_SECRET_ACCESS_KEY,
+    BUCKET_BUCKET_NAME,
+  } = getEnv()
+
+  const expiresIn = request.queryStringParameters?.expires ?
+    Number(request.pathParameters?.expires)
```

Comment:
```suggestion
    parseInt(request.queryStringParameters?.expires)
```

Reply:
Needs a test ;)

Reply:
refactored for a util so that we can test